zenith_local.md

```diff
@@ -0,0 +1,103 @@
+# Zenith local
+
+Here I list some objectives to keep in mind when discussing zenith-local design and a proposal that brings all components together.  Your comments on both parts are very welcome.
+
+#### Why do we need it?
+- For distribution - this easy to use binary will help us to build adoption among developers.
+- For internal use - to test all components together.
+
+In my understanding, we consider it to be just a mock-up version of zenith-cloud.
+> Question: How much should we care about durability and security issues for a local setup?
+
+
+#### Why is it better than a simple local postgres?
```

Comment:
I think that there will  can provide at least two more advantages comparing with vanilla Postgres:
page-level compression and encryption. We can quite easily add them, unlike standard Postgres.
Encryption requires more attention and efforts than compression because we need to concern encryption of WAL and local file cache (if any).